src/main/locales/da/locales.h

```diff
@@ -84,12 +84,17 @@
 // CLI
 #define TR_CLI_ERROR_INVALID_NAME   "UGYLDIGT NAVN: %s"
 #define TR_CLI_ERROR_MESSAGE        "%s KAN IKKE AENDRES. AKTUEL VAERDI: '%s'"
+#define TR_CLI_PARSINGFAIL          "PARSING FEJLET"
+#define TR_CLI_INVALIDRESSOURCE     "UGYLDIGT RESOURCE NAVN: '%s'"
+#define TR_CLI_ARG_INVALIDCOUNT     "UGYLDIGT ANTAL ARGUMENTER"
+#define TR_CLI_ARG_INVALIDRANGE     "%s ER IKKE MELLEM %d OG %d"
+#define TR_CLI_ARG_OUTOFRANGE       "ARGUMENT UDENFOR INTERVAL"
 
 #define TR_CLI_DSHOT_READ           "Dshot laest: %u"
 #define TR_CLI_DSHOT_INVALID_PKT    "Dshot ugyldig pakker: %u"
 #define TR_CLI_DSHOT_DIR_CHANGE     "Dshot retning  Skift  cyklus: %u, mikro:  %u"
-#define TR_CLI_DSHOT_HEAD1          "Motor    Type   eRPM    RPM     Hz Invalid   TEMP    VCC   CURR  ST/EV   DBG1   DBG2   DBG3"
-#define TR_CLI_DSHOT_LINE1          "=====  ====== ====== ====== ====== ======= ====== ====== ====== ====== ====== ====== ======"
+#define TR_CLI_DSHOT_HEAD1          "Motor    Type   eRPM    RPM     Hz    Invalid   TEMP    VCC   CURR  ST/EV   DBG1   DBG2   DBG3"
+#define TR_CLI_DSHOT_LINE1          "=====  ====== ====== ====== ====== ========== ====== ====== ====== ====== ====== ====== ======"
```

Comment:
Is invalid 3 characters more in other languages?

Reply:
No, not the headline, but the text " NO DATA" (8) to "INGEN DATA) (11)
![image](https://user-images.githubusercontent.com/99370924/219906756-f6257279-5658-42f0-b61d-930e43aa0cce.png)

Reply:
:)